src/direct-auth-idx/direct-auth-idx/Helpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNet.Identity;
+using Okta.Idx.Sdk;
+using System.Security.Claims;
+
+namespace direct_auth_idx
+{
+    public static class Helpers
```

Comment:
nit: Let's rename this class to something like IdentityHelper or AuthenticationHelper.